src/ZfcUser/Authentication/Adapter/Db.php

```diff
@@ -12,7 +12,7 @@
 use ZfcUser\Mapper\User as UserMapperInterface;
 use ZfcUser\Options\AuthenticationOptionsInterface;
 
-class Db extends AbstractAdapter implements ServiceManagerAwareInterface
```

Comment:
removing `ServiceManagerAwareInterface` is BC break what if someone extended this and have their own factory but depend on initializers to add serviceManager.